docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -123,65 +125,52 @@ image::connectors-diagram.png[Illustration of data flow from data source through
 endif::[]
 
 [id="proc-configuring-kafka-for-connectors_{context}"]
-== Configuring the {product-kafka} instance for use with {product-long-connectors}
+== Verifying that you have the prerequisites for using {product-long-connectors}
```

Comment:
According to the modular doc standards, this section should just be called, "Prerequisites", since it lists the prereqs for the entire assembly.

Reply:
I understand - This is a new thing we are trying - the PM requested that we add this verification to make sure that the user understands the prereq. We are still in Service Preview and want to try this out.